code/__HELPERS/icons.dm

```diff
@@ -975,4 +975,32 @@ obj/proc/apply_overlay(var/icon = null, var/atom/A, var/icon_state = null, var/p
 		return icon2html(thing, target)
 
 	var/icon/I = getFlatIcon(thing)
-	return icon2html(I, target)
\ No newline at end of file
+	return icon2html(I, target)
+
+
+//For creating consistent icons for human looking simple animals
+/proc/get_flat_human_icon(icon_id, datum/job/J, datum/preferences/prefs, dummy_key, showDirs = GLOB.cardinals, outfit_override = null)
+	var/static/list/humanoid_icon_cache = list()
+	if(!icon_id || !humanoid_icon_cache[icon_id])
+		var/mob/living/carbon/human/dummy/body = generate_or_wait_for_human_dummy(dummy_key)
+
+		if(prefs)
+			prefs.copy_to(body,TRUE,FALSE)
+		if(J)
+			J.equip(body, TRUE, FALSE, outfit_override = outfit_override)
```

Comment:
```suggestion
			J?.equip(body, TRUE, FALSE, outfit_override = outfit_override)
```
And remove the `if` check.
This will conflict with my icons PR, though. :cry: